application/ui/src/app/services/users.service.ts

```diff
@@ -141,6 +141,22 @@ export class UsersService {
   }
 
   sendEmail(matches: Matching[]) {
+    return this.getOrganizations().pipe(
+      map((orgs) => {
+        let orga = "";
+        if (orgs && orgs.length > 0) {
+          orga = orgs[0];
+        }
+        return orga;
+      }),
+      switchMap((orga) =>
+        this.http.post(`${this.urlApi}/email-matches`, { matches, orga })
+      ),
+      shareReplay(1)
+    );
+  }
+
+  sendEmails(matches: Matching[]){
```

Comment:
why do you create this function is it used somewhere? can you remove this sendEmails not using organization?